subprojects/packagefiles/libwebsockets/meson.build

```
@@ -907,7 +907,9 @@ libwebsockets = library(
   ],
 )
 
-depinc = include_directories('include')
+depinc = []
+depinc += include_directories('include')
+depinc += include_directories('.')
```

It'd be a bit cleaner to do:
```suggestion
depinc = include_directories('include', '.')
```